src/main/resources/templates/teams.html

```diff
@@ -7,16 +7,35 @@
     </head>
     <body class="mybg loginbackground">
         <div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+        <p>&nbsp;</p>
+        <p>&nbsp;</p>
+        <p>&nbsp;</p>
         <div class="container">
+            <p>&nbsp;</p>
+            <!--<div class="sticky-container">-->
+                <!--<p>&nbsp;</p>-->
+                <!--<ul class="sticky">-->
+                    <!--<li>-->
+                        <!--<p>&nbsp;&nbsp;&nbsp;Your Teams</p>-->
+                    <!--</li>-->
+                    <!--<li>-->
+                        <!--<p>&nbsp;&nbsp; Requested Teams</p>-->
+                    <!--</li>-->
+                    <!--<li>-->
+                        <!--<p>&nbsp;&nbsp;&nbsp;Participated Teams</p>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)